### ISSUE TITLE
Not working with Django 1.11

### ISSUE THREAD

**Issue by jungornti** [shtalinberg/django-el-pagination#74]
When trying to use pagination, i got this error: `context must be a dict rather than Context.`

This exception is raised on line 192:

```
/home/asyncee/git/afisha/env/lib/python3.6/site-packages/el_pagination/models.py in __unicode__
                elif item == 'last':
                    pages.append(self.last_as_arrow())
                else:
                    pages.append(self[item])
            context = Context(self.context)
            context.update({'pages': pages})
            return loader.render_to_string('el_pagination/show_pages.html',
                                           context) ← this line
        return ''
```
And the context looks like list:
```
callable_or_path	None
context	[{'True': True, 'False': False, 'None': None}, [{'True': True, 'False': False, 'None': None}, {}, {}, {'event_type': ..., 'events': '...'}]>, {'block': <Block Node: content. Contents: [<TextNode: '\n\n    '>, <el_pagination.templatetags.el_pagination_tags.PaginateNode object at 0x7f18a1bbd6d8>, <TextNode: '\n\n    <ul>\n        '>, <For Node: for event in events, tail_len: 11>, <TextNode: '\n    </ul>\n\n    '>, <el_pagination.templatetags.el_pagination_tags.ShowPagesNode object at 0x7f18a1bbd748>, <TextNode: '\n\n'>]>}], {'pages': [<el_pagination.models.ELPage object at 0x7f18a040ea90>, <el_pagination.models.ELPage object at 0x7f18a040ef98>, <el_pagination.models.ELPage object at 0x7f18a040e208>, None, <el_pagination.models.ELPage object at 0x7f18a040eb38>, <el_pagination.models.ELPage object at 0x7f18a040ec18>, <el_pagination.models.ELPage object at 0x7f18a040ec88>, <el_pagination.models.ELPage object at 0x7f18a040ecf8>]}]
item	'next'
pages    [<el_pagination.models.ELPage object at 0x7f18a040ea90>,
             <el_pagination.models.ELPage object at 0x7f18a040ef98>,
             <el_pagination.models.ELPage object at 0x7f18a040e208>,
             None,
             <el_pagination.models.ELPage object at 0x7f18a040eb38>]
pages_callable	<function get_page_numbers at 0x7f18a3750bf8>
self	<el_pagination.models.PageList object at 0x7f18a00e5da0>
```

**Comment by shtalinberg** [comment]
I'll find the time and make a release before the end of this week to fix it

**Comment by jungornti** [comment]
It would be very nice, thank you!

**Comment by shtalinberg** [comment]
Django 1.11 have bug with Pagination class 
https://github.com/django/django/commit/c4980e28e57f385d8ffed5e32ce373e52ce61049
I will check answer to my question

**Comment by jungornti** [comment]
Thank you for investigating into issue, hope for fast resolution.

**Comment by shtalinberg** [comment]
yes, now it resolved and I make changes for release 3.1.0 today
